src/iframe-helper.js

```diff
@@ -136,6 +137,9 @@ function getListenForEvents(parentWin, sentinel, origin, triggerWin) {
       // 3P code path, we may accept messages from nested frames.
       windowEvents = we;
       break;
+    } else {
+      windowEvents = we;
+      break;
```

Comment:
got rid of the `if sentinel = 'amp'` branch
This has to change because we are actually passing the sentinel around for a4a frames now and need all the listeners to work with these frames. As it was, the a4a frame was not working correctly.